components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -98,6 +98,12 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, nullable) UIView *titleView;
 
+/**
+ The font applied to the title of navigation bar. This font is set through UILabel Font property.
```

Comment:
Why do we have a separate API if clients should "set through UILabel Font property"? Can we not just expect them to set that property instead?